### DISCUSSION TITLE
Apache no longer starts after editing vhosts file

### DISCUSSION THREAD

**Discussion by Omoioma** [Rehike/Rehike#375] [Q&A] [answered]
This is more of a XAMPP issue but I will still post here.

I tried to edit the httpd-vhosts.conf and add the following code:
##<VirtualHost 127.0.0.1:443>
   ##ServerName www.youtube.com
   ##DocumentRoot "C:/xampp/htdocs"
   ##SSLEngine on
   ##SSLCertificateFile "conf/ssl.crt/server.crt"
   ##SSLCertificateKeyFile "conf/ssl.key/server.key"
   ##<Directory "C:/xampp/htdocs">
      AllowOverride All
      Require all granted
   ##</Directory>
##</VirtualHost>

But XAMPP gives me this error. 

8:39:09 AM  [Apache] 	Error: Apache shutdown unexpectedly.
8:39:09 AM  [Apache] 	This may be due to a blocked port, missing dependencies, 
8:39:09 AM  [Apache] 	improper privileges, a crash, or a shutdown by another method.
8:39:09 AM  [Apache] 	Press the Logs button to view error logs and check
8:39:09 AM  [Apache] 	the Windows Event Viewer for more clues
8:39:09 AM  [Apache] 	If you need more help, copy and post this
8:39:09 AM  [Apache] 	entire log window on the forums.

Changing the ports do not help.

**Comment by aubymori** [comment, marked as answer]
That XML is severely malformed.

Try adding this instead:
```xml
<VirtualHost 127.0.0.1:443>
   ServerName www.youtube.com
   DocumentRoot "C:/xampp/htdocs"
   SSLEngine on
   SSLCertificateFile "conf/ssl.crt/server.crt"
   SSLCertificateKeyFile "conf/ssl.key/server.key"
   <Directory "C:/xampp/htdocs">
      AllowOverride All
      Require all granted
   </Directory>
</VirtualHost>
```

**Reply by Omoioma** [reply]
I have a tutorial on how to setup Rehike with 2013 theme in English
https://www.youtube.com/watch?v=pTBgQFpeDYA


(not the best quality but works)
